sections/Deals/AcceptOrReject/AcceptOrRejectBox/AcceptOrRejectBox.tsx

```diff
@@ -140,104 +129,119 @@ const AcceptOrRejectDealBox: FC<AcceptOrRejectDealBoxProps> = ({
 		setInputValue(maxValue);
 	};
 
+	const handleMaxWithdrawButtonClick = () => setInputValue(userPoolBalance || 0);
+
+	const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
```

Comment:
these types of changes are what I am most concerned with but as long as we are just moving around the same logic that is fine